### PR TITLE
Frontend Install Doc Update

### PR DESCRIPTION
**Pull request by Ap0c**
## What does this change?

Tweaks the docs for installing frontend on a dev machine, adding a little extra information.
## What is the value of this and can you measure success?

It helps to avoid a couple of pitfalls that might crop up when building on a dev machine for the first time.

### CODE REVIEW

**Comment by katebee** [comment]
👍

File: docs/01-start-here/01-installation-steps.md [hunk 1]
```diff
@@ -56,7 +58,7 @@ Containers can be thrown away very easily. To do so:
 ## Local machine
 You need a Mac or Linux PC (Ubuntu).
 
-Before checking out the repository you may need to [add an SSH key to your GitHub account](https://help.github.com/articles/generating-ssh-keys/). Before pushing changes you may need to [create an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
```
Comment (NataliaLKB) [inline docs/01-start-here/01-installation-steps.md]: Is this okay to remove?

**Comment by NataliaLKB** [comment]
One question, but looks great! 👍

File: docs/01-start-here/01-installation-steps.md [hunk 2]
```diff
@@ -18,12 +18,14 @@ All development laptops must be encrypted. If you are not 100% sure, please ask
...
 2. Make your change and push to a new branch (you can review the closed PRs for help)
 3. Submit a PR
 
 # Local Test Server setup
 
+You can either set up the frontend using a Docker container, or through a local install on your machine. Before checking out the repository you may need to [add an SSH key to your GitHub account](https://help.github.com/articles/generating-ssh-keys/). Before pushing changes you may need to [create an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
+
 ## Docker
 
 Prerequesites:
```
Comment (jfsoul) [inline docs/01-start-here/01-installation-steps.md]: I can't unsee this typo 👀

File: docs/01-start-here/01-installation-steps.md (hunk 1 above)
Comment (Ap0c) [inline docs/01-start-here/01-installation-steps.md]: I've shifted that up to the start of the enclosing section 'Local Test Server setup', as I think it should probably apply to the Docker instructions too?

File: docs/01-start-here/01-installation-steps.md [hunk 3]
```diff
@@ -18,12 +18,14 @@ All development laptops must be encrypted. If you are not 100% sure, please ask
...
 
 # Obtain AWS credentials
-1. Fork [Janus](https://github.com/guardian/janus) and follow the readme
+1. Fork [Janus](https://github.com/guardian/janus) and follow the readme (**Note:** you will need permission to access the Janus repo to do this, and 2FA set up on your Google account)
 2. Make your change and push to a new branch (you can review the closed PRs for help)
 3. Submit a PR
 
 # Local Test Server setup
 
+You can either set up the frontend using a Docker container, or through a local install on your machine. Before checking out the repository you may need to [add an SSH key to your GitHub account](https://help.github.com/articles/generating-ssh-keys/). Before pushing changes you may need to [create an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
```
Comment (jfsoul) [inline docs/01-start-here/01-installation-steps.md]: @TBonnin we should probably adding a cautionary message re: docker here?

(not for this PR though probably)

File: docs/01-start-here/01-installation-steps.md (hunk 2 above)
Comment (Ap0c) [inline docs/01-start-here/01-installation-steps.md]: "Prerequesites"?

Comment (Ap0c) [inline docs/01-start-here/01-installation-steps.md]: Fixed.

**Comment by jfsoul** [comment]
Thanks! 👏

File: docs/01-start-here/01-installation-steps.md (hunk 1 above)
Comment (NataliaLKB) [inline docs/01-start-here/01-installation-steps.md]: 👍

File: docs/01-start-here/01-installation-steps.md (hunk 3 above)
Comment (TBonnin) [inline docs/01-start-here/01-installation-steps.md]: Yep. Will do

**Pull request merged**